### ISSUE TITLE
[Feature]: Create a Settings table.

### ISSUE THREAD

**Issue by pboers1988** [workfloworchestrator/orchestrator-core#417]
### What should we build?

We have quite a few settings that need to be managed for the api. Things that come to mind are whether Oauth2 is enabled or what the executor is. Furthermore things like translations might also be able to be dynamically managed when they are stored in a database table. Please design and implement a mechanism through which we can dynamically manage all settings.

Please define the following endpoints:

- Get endpoint where the key value pairs of each setting can be retrieved
- Get endpoint where the key value pairs of each translation can be retrieved
- Patch endpoint where the key value pairs of each setting can be changed
- Patch endpoint where the key value pairs of each translation can be changed

Make sure that all settings that no longer are managed in the database are loaded from that database.

Once this has been implemented, please create a frontend page to manage these endpoints.

**Comment by pboers1988** [comment]
Perhaps the first step should be to report what settings are in use and how they are configured in an admin page.

**Comment by Mark90** [comment]
I agree a setting table would be nice to manage runtime configurable settings. But the settings mentioned here (oauth2 being enabled, which worker executor to use) are not runtime configurable, as they require a restart of the orchestrator api/worker processes. And for at least OAUTH2_ENABLED there is also the question of whether the DB value should take precedence over the env-var value. 
Should this feature include those settings as well, or just the "easier" manageable settings?
